### DISCUSSION TITLE
Google Why we’re committing $10 billion to advance cybersecurity

### DISCUSSION THREAD

**Discussion by wangleo61** [kaiyuanshe/ONES#57] [日常讨论]
Google Why we’re committing $10 billion to advance cybersecurity

https://blog.google/technology/safety-security/why-were-committing-10-billion-to-advance-cybersecurity/

An update on the threat landscape
https://blog.google/threat-analysis-group/update-threat-landscape-ukraine/

SLSA
https://security.googleblog.com/2021/06/introducing-slsa-end-to-end-framework.html

**Comment by will-ww** [comment]
Good sharing~

**Comment by wangleo61** [comment]
The ecosystem landscape become more clearly, Google->Investment Security->Research and LF OpenSSF ->Go language ->K8s

**Reply by will-ww** [reply]
> The ecosystem landscape become more clearly, Google->Investment Security->Research and LF OpenSSF ->Go language ->K8s

这个链路厉害！
